### ISSUE TITLE
Flint build creates empty directories that mirror your flint project's directories.

### ISSUE THREAD

**Issue by AriaFallah** [flintjs/flint#302]
As far as I know flint build used to generate a structure like this:

```bash
build
|-- _
|-- LICENSE
|-- README
|-- index.html
```

but now in addition to those it's creating all the same directories that are in my flint directory, but with nothing in them.

```bash
build
|-- _
|-- LICENSE
|-- README
|-- index.html
|-- actions
|-- components
|-- containers
|-- reducers
|-- sagas
|-- store
|-- services
|-- store
|-- views
```

This appeared at some point during 1.5. Maybe when `flint build --cached` was introduced?

**Comment by natew** [comment]
We moved to gulp for a few more things in 1.5, need to check how to avoid this.

**Comment by natew** [comment]
Fixed in 1.5.21
